src/styles/header.scss

```
@@ -0,0 +1,59 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  justify-content: flex-end;
+  margin: 0;
+  padding-left: 0;
+  text-transform: uppercase;
+}
+
+.nav__item {
+  margin: auto 0 auto 20px;
```

```suggestion
  margin-left: 20px;
```
the same result, but simpler (easier to get how it will look like from the code)